src/odemis/driver/test/orsay_test.py

```diff
@@ -2057,46 +2060,55 @@ def connector_test(test_case, va, parameters, valuepairs, readonly=False, hw_saf
         for i in range(len(parameters)):
             for a in attributes:
                 setattr(parameters[i], a, par_value1[i])
+        # TODO to consistently pass the some tests an update of all the VAs in the orsay server is added. This
+        #  suggests that the subscription doesn't work properly. Maybe this should be checked or the update_VAs
+        #  should be runned in a thread continuously
+        test_case.oserver.update_VAs()  # Ensure the VAs will be updated
```

Comment:
Change TODO such that the default is to not ensure updating and then uncomment updating line whenever development/debugging